src/components/notes/AddNote.js

```diff
@@ -0,0 +1,116 @@
+import { TextField, Card, CardActions, Button, useTheme, Divider, Typography } from '@material-ui/core'
+import {MarkdownEditor} from '../commons/MarkdownEditor'
+import {useState} from 'react'
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import { TagsList } from './TagsList'
+import '../../App.css';
+import { useDevMail } from '../../utils/hooks/useDevMail';
+import { encodeBase64 } from '../../utils/hooks/Base64';
+import { useDevPassword } from '../../utils/hooks/useDevPassword';
+import { NoteApi } from '../../utils/apis/NoteApi';
+import Swal from 'sweetalert2'
+import {withSwal} from '../commons/formsCommons/WithSwal'
+
+export const AddNote = ({ onCancel, uuid, uuid_key, setReload }) => {
+
+    const theme = useTheme()
+    const {getEncoded} = useDevPassword()
+
+    const [noteText, setNoteText] = useState('')
+    const [tags, setTags] = useState([])
+    const [currentTag, setCurrentTag] = useState('')
+
+    const {get, set} = useDevMail()
+
+    const handleNoteTextChange = ({html, text}) => {
+        setNoteText(text)
+    }
+
+    const handleAddCurrentTag = () => {
```

Comment:
Skąd to się tu wzięło. Czy to nie miało być w ES-246/ES-233?

Reply:
Nie wiem jak Ty się tu doklikałeś, ale znowu do starego diffa zrobiłeś review xDD